### DISCUSSION TITLE
Add --bc1 --bc2 and --bc3 to allow variation in content color with various contract against the background color

### DISCUSSION THREAD

**Discussion by wighawag** [saadeghi/daisyui#1614] [Ideas / feature requests]
Currently we have `--b1` `--b2` `--b3` but we only have one `--bc`

I would like to get a slightly less contrasted text color against my background but my only way currently is to hardcode a value and this then break when I switch from a dark / light theme

**Comment by saadeghi** [comment]
Using opacity like this can give you the same result:
```
text-base-content/70
text-base-content/50
```

It will use `--bc` with less opacity which looks the same if we lower the brightness
